package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever-components",
-  "version": "0.11.0",
+  "version": "0.12.0",
```

Comment:
for pre-1.0 releases, a minor bump counts as a major rev, since 0.x releases are considered unstable.
patch bumps become the minor bump in that case